task1/task1.html

```diff
@@ -0,0 +1,167 @@
+                </td>
+                <td>
+                    <table>
+                        <thead align=center>
+                            <tr>
+                                <th>
+                                    <font size="5">
+                                        Main content
+                                    </font>
+                                </th>
+                            </tr>
+                        </thead>
+                        <tbody>
+
+                            <tr>
+                                <td>
+                                    <picture>
+                                        <img src="snowdog.jpg" alt="Dog in the snow" height="100" width="150" align=right>
+                                    </picture>
+                                    It has long been established that in assessing the design and composition of readable text interferes with concentration. Lorem Ipsum is used because it provides a more or less standard template filling, as well as a real distribution of letters and spaces
+                                    in paragraphs, which is not obtained by simple duplication "Here is your text.. Here your text.. Here your text.."Many of the programs electronic layout and HTML editors use Lorem Ipsum as default text, so that keyword
+                                    search "lorem ipsum" immediately shows how many web pages are still waiting for his present birth. Over the years, the text Lorem Ipsum has received a lot of versions. Some versions appeared by mistake, some - intentionally
+                                    (for example, humorous versions).It has long been established that in assessing the design and composition of readable text interferes with concentration. Lorem Ipsum is used because it provides a more or less standard
+                                    template filling, as well as a real distribution of letters and spaces in paragraphs, which is not obtained by simple duplication "Here is your text.. Here your text.. Here your text.."Many of the programs electronic
+                                    layout and HTML editors use Lorem Ipsum as default text, so that keyword search "lorem ipsum" immediately shows how many web pages are still waiting for his present birth. Over the years, the text Lorem Ipsum has received
+                                    a lot of versions. Some versions appeared by mistake, some - intentionally (for example, humorous versions).<sup><font size=1>1</font></sup>
+                                </td>
+                            </tr>
+                            <tr align=center>
+                                <td>
+                                    &bull; <i>Italic item<sup><a href="#"><font size=1>2</font></a></sup></i>
+                                </td>
+                            </tr>
+                            <tr align=center>
+                                <td>&bull; <b>Bold item</b></td>
```

Comment:
ul + li?

Reply:
I haven't found a way to align ul bullets without any CSS just next to list items when tr align=center, they appear to far on the left side of text